mRemoteV1/Connection/Protocol/Http/Connection.Protocol.HTTPBase.cs

```diff
@@ -178,6 +181,15 @@ private void wBrowser_Navigated(object sender, WebBrowserNavigatedEventArgs e)
 			objWebBrowser.Navigated -= wBrowser_Navigated;
 		}
 
+        //Refernce: https://bitbucket.org/geckofx/geckofx-33.0/issues/90/invalid-security-certificate-error-on
+        private void GeckoBrowser_ValidityOverride(object sender, CertOverrideEventArgs e)
+        {
+            if(InterfaceControl.Info.HttpsTrustInsecureCerts == false) return;
```

Comment:
We could make the prompt optional. Shown by default, but include an 'always accept certs' sort of check box. Best method for implementing might be to request a class that can handle the override via property injection.